adops/setting-up-prebid-native-in-dfp.md

```diff
@@ -21,34 +21,33 @@ For more information about DFP native ad setup, see the [DFP native ads document
 * TOC
 {:toc}
 
-## 1. Add a native ad unit
+## 1. Create a native ad unit
 
-You can specify a size for the ad unit, or specify the "fluid" size.  In this case we'll go with **Fluid**.
+Under **Display > Native**, select the **HTML & CSS editor** option.
 
-{: .pb-img.pb-md-img :}
-![prebid native adunit]({{site.github.url}}/assets/images/ad-ops/dfp-native/prebid_native_adunit.png)
+## 2. Define ad settings
 
-## 2. Create a custom native ad format
+For **Ad size** you can specify a specific size for the ad unit or specify the "fluid" size.  In this case we'll go with **Fluid**.
 
-For Prebid, create a [custom native ad format](https://support.google.com/dfp_sb/answer/6366911?hl=en).
+Select **New format** under **Custom format**. (If you've already created an ad unit with the format you want, you can select **Existing format** and select the format to apply to this ad unit.)
 
-Make sure to remove all variables except **Click-through URL**.
+Click **Add Variable**. Use the **File** type and create a **ClickthroughURL** variable.
```

Comment:
Is this still required (looks like it may not be in the new Native UI)? If so, why is this a "file" type as opposed to a "URL" type?

Reply:
I don't believe this step is needed anymore.